base/operators.jl

```diff
@@ -230,7 +230,7 @@ end
 
 Unsigned right bit shift operator, `x >>> n`. For `n >= 0`, the result is `x`
 shifted right by `n` bits, where `n >= 0`, filling with `0`s. For `n < 0`, this
-is equivalent to `x [<<](:func:`<<`) -n`].
+is equivalent to `x << -n`.
```

Comment:
It does, though the idea behind it, adding links in bits of code, would be great to have at some point. Allowing Documenter to automatically add links inside literals and code blocks should probably be possible I think, though would need a bit of work.